src/pages/AnimeFenix/meta.json

```diff
@@ -0,0 +1,7 @@
+{
+  "search": "*://*.animefenix.tv/animes?q={searchtermPlus}",
```

Comment:
Please use a url without wildcards, as it is to generate an search url and not to match one